technologies/job/aws-lambda/functions/instanceActions.js

```diff
@@ -0,0 +1,184 @@
+const { Response, JobStatus, Log, Stream } = require('@saagie/sdk');
+const AWS = require('aws-sdk');
+
+/**
+ * Logic to start the external job instance.
+ * @param {Object} params
+ * @param {Object} params.job - Contains job data including featuresValues.
+ * @param {Object} params.instance - Contains instance data.
+ */
+exports.start = async ({ job, instance }) => {
+  try {
+    console.log('START INSTANCE:', instance);
+    AWS.config.update({credentials: { accessKeyId : job.featuresValues.endpoint.aws_access_key_id, secretAccessKey:  job.featuresValues.endpoint.aws_secret_access_key}});
+    AWS.config.update({region: job.featuresValues.endpoint.region});
+
+    const lambda = new AWS.Lambda({apiVersion: '2015-03-31'});
```

Comment:
```suggestion

const AWS_LAMBDA_OPTIONS = { apiVersion: '2015-03-31' };

/**
 * Logic to start the external job instance.
 * @param {Object} params
 * @param {Object} params.job - Contains job data including featuresValues.
 * @param {Object} params.instance - Contains instance data.
 */
exports.start = async ({ job, instance }) => {
  try {
    console.log('START INSTANCE:', instance);
    AWS.config.update({credentials: { accessKeyId : job.featuresValues.endpoint.aws_access_key_id, secretAccessKey:  job.featuresValues.endpoint.aws_secret_access_key}});
    AWS.config.update({region: job.featuresValues.endpoint.region});

    const lambda = new AWS.Lambda(AWS_LAMBDA_OPTIONS);
```

One thing that can be done here is to declare the `AWS.Lambda` options as a global variable so it can be re-used in every functions. It's better for maintainability.